Source/Utils/WarpXUtil.H

```diff
@@ -141,4 +141,23 @@ void AlwaysAssert(bool is_expression_true, const std::string& msg);
 
 }
 
+namespace WarpXUtilStr
+{
+    /** Return true if elem is in vect, false otherwise
+     * @param[in] vect vector of strings, typically names
+     * @param[in] elem single string
+     * @return true if elem is in vect, false otherwise
+     */
+    bool is_in(std::vector<std::string> vect,
```

Comment:
Probably better to pass these by const reference.

Reply:
Done, thanks!